internal/tools/pipeline/dbclient/op_pipeline_definition.go

```diff
@@ -27,7 +27,7 @@ func (client *Client) GetPipelineDefinition(id string, ops ...SessionOption) (*d
 	var pipelineDefinition db.PipelineDefinition
 	var has bool
 	var err error
-	if has, _, err = session.Where("id = ? and soft_deleted_at = 0", id).GetFirst(&pipelineDefinition).GetResult(); err != nil {
+	if has, err = session.Where("id = ? and soft_deleted_at = 0", id).Get(&pipelineDefinition); err != nil {
```

Comment:
How about there are multiple records?

Reply:
The `GetFirst` is a wrapper for the `Get`.
`xorm` does not support `GetFirst`, and `Get` is used to get single data entries, while the `Find` is used for multiple records.
So, replace `GetFirst` as `Get` directly.
![image](https://github.com/erda-project/erda/assets/86157286/47c0050a-dafc-4509-9bdd-50879ec9e948)

<img width="1185" alt="image" src="https://github.com/erda-project/erda/assets/86157286/06f69010-6ffc-4e31-afbe-f2c2d155723d">